### PR TITLE
[DC-836] [DC-1194] Update Person Table Repopulation CR to New Base Class

### PR DESCRIPTION
**Pull request by ksdkalluri**
* Refactors repopulate_person_post_deid CR to use base class
* adds unittests
* Fixes integration test

Signed-off-by: Krishna Kalluri <krishnasaikalluri@gmail.com>

### CODE REVIEW

File: data_steward/cdr_cleaner/cleaning_rules/repopulate_person_post_deid.py [hunk 1]
```diff
@@ -25,114 +25,154 @@
 """
 import logging
```
Comment (amcgrenera-vumc) [inline data_steward/cdr_cleaner/cleaning_rules/repopulate_person_post_deid.py:27]: Maybe use the new pipeline logging?

```
from utils import pipeline_logging
```

File: data_steward/cdr_cleaner/cleaning_rules/repopulate_person_post_deid.py [hunk 2]
```diff
@@ -206,13 +283,14 @@ def get_repopulate_person_post_deid_queries(project_id,
```
Comment (amcgrenera-vumc) [inline data_steward/cdr_cleaner/cleaning_rules/repopulate_person_post_deid.py:283]: if using new `pipeline_logging` you can configure here, something like:

```
pipeline_logging.configure(level=logging.DEBUG, add_console_handler=ARGS.console_log)
```

Comment (ksdkalluri) [reply data_steward/cdr_cleaner/cleaning_rules/repopulate_person_post_deid.py:283]: I used logging as all the cleaning rules are using this. I have no specific preference to use one or the other I can update this if this is required to be updated.

Comment (amcgrenera-vumc) [reply data_steward/cdr_cleaner/cleaning_rules/repopulate_person_post_deid.py:283]: No need to update now, as we discussed I'll bring up in eng sync!

**Comment by amcgrenera-vumc** [comment]
@ksdkalluri this looks good to me! Tests pass locally and meets all acceptance criteria.

**Review by amcgrenera-vumc — APPROVED** [review]

**Pull request merged**